Tools/autotest/ArduPlane_Tests/EKFlaneswitch/CMAC-circuit.txt

```diff
@@ -0,0 +1,9 @@
+QGC WPL 110
```

Comment:
I think just using TAKEOFF mode may be simpler than adding these missions

Reply:
removed the mission and now run the test while plane is in circle mode